modules/nf-core/annotsv/annotsv/main.nf

```diff
@@ -0,0 +1,70 @@
+process ANNOTSV_ANNOTSV {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::annotsv=3.3.4"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/annotsv:3.3.4--py311hdfd78af_0' :
+        'quay.io/biocontainers/annotsv:3.3.4--py311hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(variants), path(variants_index)
+    tuple val(meta2), path(annotations)
+    tuple val(meta3), path(candidate_genes)
+    tuple val(meta4), path(candidate_small_variants)
+    tuple val(meta5), path(false_positive_snv)
+    tuple val(meta6), path(gene_transcripts)
+
+    output:
+    tuple val(meta), path("*.tsv")              , emit: tsv
+    tuple val(meta), path("*.unannotated.tsv")  , emit: unannotated_tsv, optional:true
+    tuple val(meta), path("*.vcf")              , emit: vcf, optional:true
```

Comment:
Line the optional up? I don't know what has been done for other modules...

Reply:
I think only the emits are aligned (I prefer this too as it will become just some scattered text this way)